docs/en/05_Contributing/02_Request_for_comment.md

```diff
@@ -0,0 +1,52 @@
+# Request for comment (RFC)
+
+## Why RFCs?
+This is part of the SilverStripe core decision-making process and addresses the gap between the idea phase and the pull request submission (and merge).
+
+The rationale behind this process is to:
+ * Encourage visibility on decision making
+ * Clarity on what the proposal is, its rationale and impact
```

Comment:
Should be: *Clarify what the proposal is, its rationale and impact* or *Provide clarity on what the proposal is, its rationale and impact*